.github/workflows/build.yml

```diff
@@ -11,7 +11,8 @@ jobs:
       fail-fast: false
       matrix:
         rails-version: [6, 7]
-        ruby-version: [2.7, 3.0, 3.1]
+        # Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.0'
+        ruby-version: ['2.7', '3.0', '3.1']
```

Comment:
Example of a Ruby 3.0 job that actually uses 3.1.2: https://github.com/cipherstash/activestash/runs/7717469076?check_suite_focus=true#step:3:22

Reply:
I absolutely fucking _adore_ YAML.  Total galaxy brain stuff.